### DISCUSSION TITLE
Error while checking account

### DISCUSSION THREAD

**Discussion by xtourtemyx** [AsamK/signal-cli#1425] [Q&A]
"Error while checking account +***********: signal-cli version is too old for the Signal-Server, please update."

all packages already updated, how i can update signal-cli at another method?

**Comment by m-ueberall** [comment]
Unless you wait for the next release which should be available soon, your only option – *not that I'd recommend it!* – would be to use a current snapshot ([see the artefacts attached to this run](https://github.com/AsamK/signal-cli/actions/runs/7688683073)).

**Reply by fireflystorm** [reply]
Does this mean that even the latest version 0.12.7 would currently not work?

**Reply by m-ueberall** [reply]
No, it does not (works for me). [OP clarified that s/he's currently using signal-cli v0.11.7.](https://github.com/AsamK/signal-cli/discussions/1425#discussioncomment-8299151)

**Reply by fireflystorm** [reply]
Got it thank you for the confirmation @m-ueberall

**Comment by xtourtemyx** [comment]
it only one method? 
i want update signal-cli to 0.12.7, right now i have signal-cli 0.11.7
can't find any guide, how to update signal-cli

**Reply by AsamK** [reply]
That depends on how you installed signal-cli
https://github.com/AsamK/signal-cli/wiki/Binary-distributions

**Comment by AugustaCharis** [comment]
Hello - I was searching the forum and found this thread, which is unfortunately already a year old.
I use signal-cli to receive Signal messages via signal-desktop.
I don't use Signal on a smartphone.
I had the problem that Signal said "Your account will be deleted soon - if you don't open Signal on your phone - which apparently has something to do with the PQXDH protocol."
- the tip:
signal-cli -a +49XXX ​​receive
led to the following response:
INFO AccountHelper - The Signal protocol expects that incoming messages are regularly received.
Error while checking account +491782105337: signal-cli version is too old for the Signal server, please update.

Hence my questions:
signal-cli --version : signal-cli 0.11.6

1. Which version of signal-cli should I use?

2. Can I update signal-cli without having to re-register?

My system:
Debian 6.1.85-1 (2024-04-11) x86_64 GNU/Linux

Thanks Stefanie (charis)
